packages/front-end/components/Experiment/TabbedPage/SetupTabOverview.tsx

```diff
@@ -150,6 +153,19 @@ export default function SetupTabOverview({
                 and parameters of your experiment
               </Box>
             )}
+            {!customFields.length ? (
```

Comment:
I think we should hide this if the description is empty.  If someone isn't even putting things in the description, they probably don't care about custom fields.